ProductWarehouse.API/Mapping/AutoMapperProfile.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AutoMapper;
+using ProductWarehouse.API.Models.Requests;
+using ProductWarehouse.API.Models.Responses;
+using ProductWarehouse.Application.Features.Queries.GetProducts;
+using ProductWarehouse.Application.Models;
+
+namespace ProductWarehouse.API.Mapping;
+
+public class AutoMapperProfile : Profile
+{
+    public AutoMapperProfile()
+    {
+        CreateMap<ProductDto, ProductResponse>();
+        CreateMap<FilterProductsRequest, ProductsQuery>();
```

Comment:
There are two options for how you can do it more organized here.
1. The approach is to map 2 private methods to represent the mapping direction.
```
    public MappingProfile()
    {
        MapFromRequestToQueriesOrCommands();
        MapFromDtoToResponse();
    }

    private void MapFromRequestToQueriesOrCommands()
    {
        CreateMap<ProductFilterRequest, ProductQuery>();
    }

    private void MapFromDtoToResponse()
    {
        CreateMap<ProductDto, ProductResponse>();
    }
```
2. The approach is to have 2 profiles with each direction

```
public class MapFromRequestToQueriesOrCommands : Profile { }
public class MapFromDtoToResponse : Profile {}
```
Decide what you like best and use the approach.

This helps a person to quickly orient himself in the complex structure of the project and to expand the maps more easily